node/db/src/rocks_config.rs

```diff
@@ -30,9 +31,10 @@ impl Default for RocksDbConfig {
             write_buffer_size: 256 * 1024 * 1024,
             max_open_files: 1024,
             max_background_jobs: None,
-            compaction_style: None,
+            compaction_style: Some("level".into()),
             compression_type: Some("lz4".into()),
             enable_statistics: false,
+            log_level: "debug".into(),
```

Comment:
Don't activate `debug` log by default here. What do you think of `warn`?

Reply:
That makes more sense for a default, yeah